app/services/idv/proofing_components.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module Idv
+  class ProofingComponents
+    def initialize(
+        user:,
+        idv_session:
+      )
+      @user = user
+      @idv_session = idv_session
+    end
+
+    def document_check
+      if user.establishing_in_person_enrollment || user.pending_in_person_enrollment
+        Idp::Constants::Vendors::USPS
+      elsif idv_session.remote_document_capture_complete?
+        DocAuthRouter.doc_auth_vendor(
+          discriminator: idv_session.document_capture_session_uuid,
+        )
+      end
+    end
+
+    def document_type
+      return 'state_id' if idv_session.remote_document_capture_complete?
+    end
+
+    def source_check
+      Idp::Constants::Vendors::AAMVA if idv_session.verify_info_step_complete?
+    end
+
+    def resolution_check
+      Idp::Constants::Vendors::LEXIS_NEXIS if idv_session.verify_info_step_complete?
+    end
+
+    def address_check
+      if idv_session.verify_by_mail?
+        'gpo_letter'
+      elsif idv_session.address_verification_mechanism == 'phone'
+        'lexis_nexis_address'
+      end
+    end
+
+    def liveness_check
```

Comment:
I don't necessarily think this should land with these unused methods in place, I just wanted to call out that the existing proofing_components infrastructure as some gaps (these columns exist in the DB but don't appear to be used)

Reply:
Created a coupla tickets (LG-13759 & LG-13760) around cleaning up these old columns